snapcraft/plugins/ant.py

```diff
@@ -33,11 +33,16 @@
       (list of strings)
       Run the given ant targets.
 
+    - ant-snap-channel:
```

Comment:
can we call this `ant-channel`? Just to be consistent with all the other plugins.

Reply:
Good call, fixed.